tests/manager/openAccount/managerCanOpenAccount.spec.js

```diff
@@ -28,4 +44,19 @@ Test:
 Tips:
  1. Do not rely on the customer row id for the step 13. Use the ".last()" locator to get the last row.
 */
+
+  const openAccountPage = new OpenAccountPage(page); 
+  
+  await openAccountPage.open();
+  await openAccountPage.selectCustomer(firstName + ' ' + lastName);
+  await openAccountPage.setCurrency('Dollar');
+  await openAccountPage.clickProcessButton();
+  await openAccountPage.reload();
+  await openAccountPage.clickCustomersButton();
+
+  const customersListPage = new CustomersListPage(page);
+  await customersListPage.open();
+  const fullName = firstName + ' ' + lastName + ' ' + postCode;
+
+  await customersListPage.assertCustomerRowHasAccountNumber(fullName);
```

Comment:
The `fullName` variable includes the postal code, which might not match the row's name attribute if it only contains the first and last name. Verify that the format of `fullName` matches the format used in the customer table for the row's name attribute.